store/store.go

```diff
@@ -300,6 +314,9 @@ func (s *Store[H]) HasAt(_ context.Context, height uint64) bool {
 	return height != uint64(0) && s.Height() >= height
 }
 
+// Append the given headers to the store. Real write to the disk happens
+// asynchronously and might fail without reporting error (just logging).
+// TODO(cristaloleg): add retries to the flush worker.
```

Comment:
Next PR.

Reply:
could you please create an issue and add it to the comment

Reply:
Done https://github.com/celestiaorg/go-header/issues/209 (not gonna edit comment 'cause gonna address this shortly after this PR).